app/src/test/java/com/kickstarter/viewmodels/TwoFactorViewModelTest.java

```diff
@@ -27,21 +34,20 @@ public void testTwoFactorViewModel_FormValidation() {
     intent.putExtra(IntentKey.FACEBOOK_LOGIN, false);
     intent.putExtra(IntentKey.FACEBOOK_TOKEN, "");
 
-    final TwoFactorViewModel vm = new TwoFactorViewModel(environment());
-    vm.intent(intent);
+    this.vm = new TwoFactorViewModel.ViewModel(environment());
+    this.vm.intent(intent);
 
-    final TestSubscriber<Boolean> formIsValid = new TestSubscriber<>();
-    vm.outputs.formIsValid().subscribe(formIsValid);
+    this.vm.outputs.formIsValid().subscribe(this.formIsValid);
 
-    formIsValid.assertNoValues();
+    this.formIsValid.assertNoValues();
 
-    vm.inputs.code("444444");
-    formIsValid.assertValue(true);
+    this.vm.inputs.code("444444");
+    this.formIsValid.assertValue(true);
 
-    vm.inputs.code("");
-    formIsValid.assertValues(true, false);
+    this.vm.inputs.code("");
+    this.formIsValid.assertValues(true, false);
 
-    koalaTest.assertValue("Two-factor Authentication Confirm View");
+    this.koalaTest.assertValue("Two-factor Authentication Confirm View");
```

Comment:
we've started adding these event strings to [KoalaEvent.java](https://github.com/kickstarter/android-oss/blob/master/app/src/main/java/com/kickstarter/libs/KoalaEvent.java) to avoid re-typing / pasting the same string over and over again if you'd like to add these now!